base/install_motor.sh

```diff
@@ -34,3 +35,33 @@ cd ..
 
 make -j${JOBS}
 make clean
+
+#Add pmac module
+
+cd ..
+
+download_github_module dls-controls pmac $PMAC_VERSION
+
+rm -f pmac/configure/RELEASE.local.linux-x86_64
+rm -f pmac/configure/RELEASE.linux-x86_64.Common
+rm -f pmac/configure/CONFIG_SITE.linux-x86_64.Common
+
+echo "
+BUILD_IOCS=NO
+USE_GRAPHICSMAGICK=NO
+
+SSH             = $(pkg-config --variable=prefix libssh2)
+SSH_LIB         = $(pkg-config --variable=libdir libssh2)
+SSH_INCLUDE     = -I$(pkg-config --variable=includedir libssh2)
```

Comment:
@ericonr, do you have any clue why `--cflags-only-I` wouldn't work even with variables defined in `libssh2.pc`?

I've compared the syntax along with @guirodrigueslima, but I couldn't understand what was happening. In this case, it is okay to have `includedir`, since it has a single include directory.